hackathon-website/src/Components/Items/NavbarItems.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const navbarItems = (props) =>
+  props.data.map((val, key) => (
+    <li
+      key={key}
+      onClick={() => {
+        window.location.pathname = val.link;
+      }}
+    >
+      <div className="icon">{props.iconite && val.icon}</div>
```

Comment:
Am lasat un comment in fisierul "SideDrawer.js" cu motivul pentru care ar trebui sa traduci variabila "iconite"